src/signals-win.c

```diff
@@ -483,11 +483,15 @@ void jl_install_default_signal_handlers(void)
 
 void jl_install_thread_signal_handler(jl_ptls_t ptls)
 {
-    size_t ssize = sig_stack_size;
-    void *stk = jl_malloc_stack(&ssize, NULL);
-    collect_backtrace_fiber.uc_stack.ss_sp = (void*)stk;
-    collect_backtrace_fiber.uc_stack.ss_size = ssize;
-    jl_makecontext(&collect_backtrace_fiber, start_backtrace_fiber);
-    uv_mutex_init(&backtrace_lock);
-    have_backtrace_fiber = 1;
+    if (!have_backtrace_fiber) {
```

Comment:
https://github.com/JuliaLang/julia/blob/6d678fec0ad0c78e80f357aac1b1e99ff0ff47ca/src/signals-win.c#L109

`have_backtrace_fiber` is not initialized.
Perhaps initialize with 0?

Reply:
Globals are always initialized (to 0)